src/v/kafka/protocol/schemata/create_acls_request.json

```diff
@@ -16,12 +16,15 @@
 {
   "apiKey": 30,
   "type": "request",
+  "listeners": ["zkBroker", "broker", "controller"],
   "name": "CreateAclsRequest",
   // Version 1 adds resource pattern type.
-  "validVersions": "0-1",
-  "flexibleVersions": "none",
+  // Version 2 enables flexible versions.
+  // Version 3 adds user resource type.
+  "validVersions": "0-3",
+  "flexibleVersions": "2+",
   "fields": [
-    { "name": "Creations", "type": "[]CreatableAcl", "versions": "0+", 
+    { "name": "Creations", "type": "[]CreatableAcl", "versions": "0+",
       "about": "The ACLs that we want to create.", "fields": [
       { "name": "ResourceType", "type": "int8", "versions": "0+",
         "about": "The type of the resource." },
```

Comment:
i find it confusing that given this diff shows we are moving from 0-1 valid versions to 0-3 and the comment says `// Version 3 adds user resource type.` but the diff doesn't show resource type being added. instead, it was already there!?

Reply:
Yeah that is super confusing, I guess maybe we should devote some time to understand what really necessitated that version bump. Probably part of the 100% compatibility story

Reply:
I was curious to what these version 3 bumps were about, I tracked it down to this KIP if your curious https://cwiki.apache.org/confluence/display/KAFKA/KIP-373%3A+Allow+users+to+create+delegation+tokens+for+other+users

Reply:
> I tracked the change in the schema to this KIP if you

nice thanks!